mesonbuild/interpreter.py

```diff
@@ -3761,7 +3761,7 @@ def unpack_env_kwarg(self, kwargs) -> build.EnvironmentVariables:
 
     def add_test(self, node, args, kwargs, is_base_test):
         if len(args) != 2:
-            raise InterpreterException('Incorrect number of arguments')
+            raise InterpreterException('test expects 2 arguments, {} given'.format(len(args))
```

Comment:
```suggestion
            raise InterpreterException('test expects 2 arguments, {} given'.format(len(args)))
```

You're missing a closing paren.

Reply:
Yeah I noticed, serves me right for not running the tests locally first.